From b27172de1e43ee6bd2271c0691b55954427fedec Mon Sep 17 00:00:00 2001
From: "dalon.zhang" <dalon.zhang@rock-chips.com>
Date: Fri, 24 Mar 2017 18:37:05 +0800
Subject: [PATCH] camera: rockchip: camsys version 0.0x21.0xb

some log is so boring, set print level more high.

Change-Id: I8cfe16f535718a130fa03ee7173e4ef325239e06
Signed-off-by: dalon.zhang <dalon.zhang@rock-chips.com>
---
 drivers/media/video/rk_camsys/camsys_internal.h    |  6 ++++--
 drivers/media/video/rk_camsys/camsys_marvin.c      | 14 +++++++-------
 drivers/media/video/rk_camsys/camsys_mipicsi_phy.c |  8 ++++----
 drivers/media/video/rk_camsys/camsys_soc_priv.c    |  6 +++---
 drivers/media/video/rk_camsys/camsys_soc_rk3288.c  |  2 +-
 drivers/media/video/rk_camsys/camsys_soc_rk3368.c  |  2 +-
 6 files changed, 20 insertions(+), 18 deletions(-)

diff --git a/drivers/media/video/rk_camsys/camsys_internal.h b/drivers/media/video/rk_camsys/camsys_internal.h
index 045d01f5fe83..2656b2f0989b 100644
--- a/drivers/media/video/rk_camsys/camsys_internal.h
+++ b/drivers/media/video/rk_camsys/camsys_internal.h
@@ -153,8 +153,10 @@
 	streaming again, iommu resource may be conflicted.
 *v0.0x21.0xa:
 	1) clock clk_vio0_noc would cause mipi lcdc no display on 3368h, remove it.
+*v0.0x21.0xb:
+	1) some log is boring, so set print level more high.
 */
-#define CAMSYS_DRIVER_VERSION                   KERNEL_VERSION(0, 0x21, 0xa)
+#define CAMSYS_DRIVER_VERSION                   KERNEL_VERSION(0, 0x21, 0xb)
 
 #define CAMSYS_PLATFORM_DRV_NAME                "RockChip-CamSys"
 #define CAMSYS_PLATFORM_MARVIN_NAME             "Platform_MarvinDev"
@@ -420,7 +422,7 @@ camsys_extdev_t *extdev, camsys_sysctrl_t *devctl, camsys_dev_t *camsys_dev)
 					devctl->ops, gpio->io, !gpio->active);
 			}
 		} else {
-			camsys_err("Sysctl %d failed, because gpio is NULL!",
+			camsys_trace(1, "Sysctl %d not do, because gpio is NULL",
 				devctl->ops);
 			err = -EINVAL;
 			goto end;
diff --git a/drivers/media/video/rk_camsys/camsys_marvin.c b/drivers/media/video/rk_camsys/camsys_marvin.c
index d4799695e486..d9ee6c06dcb7 100644
--- a/drivers/media/video/rk_camsys/camsys_marvin.c
+++ b/drivers/media/video/rk_camsys/camsys_marvin.c
@@ -455,10 +455,9 @@ static int camsys_mrv_drm_iommu_cb(void *ptr, camsys_sysctrl_t *devctl)
 		camsys_dev->dma_buf_cnt++;
 		camsys_trace
 			(
-			1,
-			"%s:iommu map dma_addr 0x%lx,attach %p,"
+			2,
+			"dma buf map: dma_addr 0x%lx,attach %p,"
 			"dma_buf %p,sgt %p,fd %d,buf_cnt %d",
-			__func__,
 			(unsigned long)dma_addr,
 			attach,
 			dma_buf,
@@ -483,12 +482,13 @@ static int camsys_mrv_drm_iommu_cb(void *ptr, camsys_sysctrl_t *devctl)
 		attach = camsys_dev->dma_buf[index].attach;
 		dma_buf = camsys_dev->dma_buf[index].dma_buf;
 		sgt = camsys_dev->dma_buf[index].sgt;
+		dma_addr = sg_dma_address(sgt->sgl);
 		camsys_trace
 				(
-				1,
-				"%s:iommu map ,attach %p,"
+				2,
+				"dma buf unmap: dma_addr 0x%lx,attach %p,"
 				"dma_buf %p,sgt %p,index %d",
-				__func__,
+				(unsigned long)dma_addr,
 				attach,
 				dma_buf,
 				sgt,
@@ -674,7 +674,7 @@ static int camsys_mrv_clkout_cb(void *ptr, unsigned int on, unsigned int inclk)
 		clk_set_rate(clk->cif_clk_out, inclk);
 		clk_prepare_enable(clk->cif_clk_out);
 		clk->out_on = on;
-		camsys_trace(1, "camsys %s clock out(rate: %dHz) turn on",
+		camsys_trace(1, "%s clock out(rate: %dHz) turn on",
 			dev_name(camsys_dev->miscdev.this_device),
 					inclk);
 	} else if (!on && clk->out_on) {
diff --git a/drivers/media/video/rk_camsys/camsys_mipicsi_phy.c b/drivers/media/video/rk_camsys/camsys_mipicsi_phy.c
index f946641d721b..fb5d83baf6f2 100755
--- a/drivers/media/video/rk_camsys/camsys_mipicsi_phy.c
+++ b/drivers/media/video/rk_camsys/camsys_mipicsi_phy.c
@@ -33,10 +33,10 @@ static int camsys_mipiphy_clkin_cb(void *ptr, unsigned int on)
 		}
 	}
 	if (on)
-		camsys_trace(1, "%s mipi phy clk in turn on",
+		camsys_trace(2, "%s mipiphy clk turn on",
 		dev_name(camsys_dev->miscdev.this_device));
 	else
-		camsys_trace(1, "%s mipi phy clk in turn off",
+		camsys_trace(2, "%s mipiphy clk turn off",
 		dev_name(camsys_dev->miscdev.this_device));
 
 	return 0;
@@ -270,11 +270,11 @@ struct platform_device *pdev, camsys_dev_t *camsys_dev)
 		/* get cru base */
 		node = of_parse_phandle(dev->of_node, "rockchip,cru", 0);
 		camsys_dev->rk_cru_base = (unsigned long)of_iomap(node, 0);
-		camsys_trace(1, "rk_cru_base=0x%lx", camsys_dev->rk_cru_base);
+		camsys_trace(2, "rk_cru_base=0x%lx", camsys_dev->rk_cru_base);
 		/* get grf base */
 		node = of_parse_phandle(dev->of_node, "rockchip,grf", 0);
 		camsys_dev->rk_grf_base = (unsigned long)of_iomap(node, 0);
-		camsys_trace(1, "rk_grf_base=0x%lx", camsys_dev->rk_grf_base);
+		camsys_trace(2, "rk_grf_base=0x%lx", camsys_dev->rk_grf_base);
 	}
 
 	return 0;
diff --git a/drivers/media/video/rk_camsys/camsys_soc_priv.c b/drivers/media/video/rk_camsys/camsys_soc_priv.c
index 82698a62931d..fdae02a286fa 100755
--- a/drivers/media/video/rk_camsys/camsys_soc_priv.c
+++ b/drivers/media/video/rk_camsys/camsys_soc_priv.c
@@ -32,7 +32,7 @@ int camsys_soc_init(unsigned int chip_type)
 	if (chip_type == 3368) {
 		strlcpy(camsys_soc_p->name, "camsys_rk3368", 31);
 		camsys_soc_p->soc_cfg = camsys_rk3368_cfg;
-		camsys_err("camsys_soc_init exit!");
+		camsys_trace(2, "camsys_soc_init exit!");
 	} else if (chip_type == 3288) {
 		if (cpu_is_rk3288()) {
 			strlcpy(camsys_soc_p->name, "camsys_rk3288", 31);
@@ -44,11 +44,11 @@ int camsys_soc_init(unsigned int chip_type)
 	} else if (chip_type == 3366) {
 		strlcpy(camsys_soc_p->name, "camsys_rk3366", 31);
 		camsys_soc_p->soc_cfg = camsys_rk3366_cfg;
-		camsys_err("camsys_soc_init exit!");
+		camsys_trace(2, "camsys_soc_init exit!");
 	} else if (chip_type == 3399) {
 		strlcpy(camsys_soc_p->name, "camsys_rk3399", 31);
 		camsys_soc_p->soc_cfg = camsys_rk3399_cfg;
-		camsys_err("camsys_soc_init exit!");
+		camsys_trace(2, "camsys_soc_init exit!");
 	}
 
 	return 0;
diff --git a/drivers/media/video/rk_camsys/camsys_soc_rk3288.c b/drivers/media/video/rk_camsys/camsys_soc_rk3288.c
index 8477864235fe..e48af21ce56e 100755
--- a/drivers/media/video/rk_camsys/camsys_soc_rk3288.c
+++ b/drivers/media/video/rk_camsys/camsys_soc_rk3288.c
@@ -376,7 +376,7 @@ camsys_dev_t *camsys_dev, camsys_soc_cfg_t cfg_cmd, void *cfg_para)
 			cru_writel(0x40004000, 0x1d0);
 		else
 			cru_writel(0x40000000, 0x1d0);
-			camsys_trace(1, "Isp_SoftRst: %d", reset);
+			camsys_trace(2, "Isp_SoftRst: %d", reset);
 		break;
 	}
 
diff --git a/drivers/media/video/rk_camsys/camsys_soc_rk3368.c b/drivers/media/video/rk_camsys/camsys_soc_rk3368.c
index 89fc8ef86604..a8e47a250a58 100644
--- a/drivers/media/video/rk_camsys/camsys_soc_rk3368.c
+++ b/drivers/media/video/rk_camsys/camsys_soc_rk3368.c
@@ -247,7 +247,7 @@ camsys_dev_t *camsys_dev, camsys_soc_cfg_t cfg_cmd, void *cfg_para)
 		else
 			__raw_writel(0x00, (void *)(camsys_dev->rk_isp_base +
 			MRV_AFM_BASE + VI_IRCL));
-			camsys_trace(1, "Isp self soft rst: %ld", reset);
+			camsys_trace(2, "Isp self soft rst: %ld", reset);
 			break;
 		}
 
-- 
2.35.3

